BCStudentSoftwareDevTeam/celts
# Created a page where admins can create new events

Summary:
For our issue, we had to create a webpage where admins could add new events.  We first created the UI and the webpage for the admin. Then we added some minor functionality such as datepickers and timepickers. Afterward, we reformated the UI to work with bootstrap. Then we worked on retrieving data from the UI. After that, we added validation on both the front and backend. We also added test cases. (it's a lot)

To test:
run the website the go to <yourIP>:8080/1/create_event
Try to create an event. 
hit save after leaving some fields blank to check for required fields.
Set start time to be after end time to check for an example of backend validation
try to create an event that already exists.
check the training for program toggle without checking required for the program for another example of backed validation.
(run the test cases for most of the other stuff)
